TombEngine/Objects/TR5/Object/tr5_rollingball.cpp

```diff
@@ -73,8 +75,11 @@ void RollingBallControl(short itemNumber)
 
 			if (distance < 16384)
 			{
-				Camera.bounce = -(((16384 - distance) * abs(item->Animation.Velocity.y)) / 16384);
-				SoundEffect(SFX_TR4_BOULDER_FALL, &item->Pose);
+				if (OCB != 0x1)
```

Comment:
Since other temporary OCBs might be added, it would be best to simply check the one flag instead of the whole bit field:
```
if ((item->TriggerFlags & 1) != 1) // Flag 1 = silent.
{
    // Do stuff.
}
```

Reply:
Yes thats a good idea, change it :)

Reply:
I had change it, can you review, test and merge it?